nifi-api/src/main/java/org/apache/nifi/processor/AbstractProcessor.java

```diff
@@ -21,13 +21,19 @@
 public abstract class AbstractProcessor extends AbstractSessionFactoryProcessor {
 
     @Override
+    protected void init(ProcessorInitializationContext context) {
+        super.init(context);
```

Comment:
I think there was originally some code in the method that got removed, I believe I can remove this method.